haskell-src/Concordium/Types/Queries.hs

```diff
@@ -119,7 +119,15 @@ data ConsensusStatus = ConsensusStatus
       -- Initially this is equal to 'csGenesisBlock'.
       csCurrentEraGenesisBlock :: !BlockHash,
       -- |Time when the current era started.
-      csCurrentEraGenesisTime :: !UTCTime
+      csCurrentEraGenesisTime :: !UTCTime,
+      -- |The current duration to wait before a round times out.
+      csCurrentTimeoutDuration :: !(Maybe Duration),
+      -- |The current round.
+      csCurrentRound :: !(Maybe Word64),
+      -- |The current epoch.
+      csCurrentEpoch :: !(Maybe Epoch),
+      -- |The trigger block time of the seedstate of the last finalized block.
```

Comment:
Maybe just expand a bit here, and say that this is the nominal epoch transition of the new epoch?